src/main/java/se/datasektionen/calypso/Darkmode.java

```diff
@@ -0,0 +1,24 @@
+package se.datasektionen.calypso;
+
+import org.springframework.web.client.RestTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import se.datasektionen.calypso.config.Config;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpEntity;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Darkmode {
+	private final Config config;
+
+	public Darkmode(Config config) {
+		this.config = config;
+	}
+
+	public boolean getCurrent() {
```

Comment:
I guess. If they would be running on the same server it shouldn't be a problem, since darkmode just fetches from redis (any valkey enjoyers here?) it should be fast enough, but since they're currently on servers in different countries and darkmode is (probably) proxied through cloudflare it adds some unnecessary latency.

Reply:
Fair, if it is only once per request to the events it could probably be fairly fine.

There was one place were the function was ran often, which I left a comment on, but if that is fixed it is probably more fine. After that maybe we could merge and add caching if latency turns out to be a problem,